packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1845,11 +1889,15 @@ export const components = (
             marginBlock: `-${odysseyTokens.Spacing2}`,
             marginInline: `-${odysseyTokens.Spacing2}`,
           },
+
+          ".MuiListItemSecondaryAction-root": {
+            display: "none",
+          },
```

Comment:
I think there's something you can import for these prop names.

Like `MuiListItemSecondaryAction_classes.root` or something. Can you look at other examples for how Shred handled that?